tests/fixture_plugins/pacbio_fixtures/metrics_fixtures.py

```diff
@@ -61,3 +63,17 @@ def pac_bio_polymerase_metrics() -> PolymeraseMetrics:
         PolymeraseDataAttributeIDs.LONGEST_SUBREAD_LENGTH_N50: 22250,
     }
     return PolymeraseMetrics.model_validate(data, from_attributes=True)
+
+
+@pytest.fixture
+def pac_bio_smrtlink_databases_metrics() -> SmrtlinkDatasetsMetrics:
+    data: dict[str, Any] = {
+        SmrtLinkDatabasesIDs.BIO_SAMPLE_NAME: "1247014000119",
+        SmrtLinkDatabasesIDs.CELL_ID: "EA094834",
+        SmrtLinkDatabasesIDs.CELL_INDEX: 0,
```

Comment:
What is the cell_index?

Reply:
The position of the smrt cell in the sequencing tray. It can hold 4 smart cells positioned 0, 1, 2 and 3